src/actions/hooks.js

```diff
@@ -263,6 +263,11 @@ export const useInstructorAvailability = (instructorID) => wrapUseEndpoint(
     }
 )();
 
+export const useUnpaidSessions = () => wrapUseEndpoint(
```

Comment:
To use `wrapUseEndpoint`, you wouldn't have the `() =>` here:

    export const useUnpaidSessions = wrapUseEndpoint(...);